lerna.json

```diff
@@ -3,7 +3,7 @@
     "blocks/*",
     "components/*"
   ],
-  "version": "5.10.2",
+  "version": "5.10.3-hotfix.0",
```

Comment:
ok cool, looks like lerna was used

Reply:
This was the command documented on the git docs in news theme development

Reply:
oh ok cool https://github.com/WPMedia/fusion-news-theme-blocks/blob/stable/News%20Theme%20Development.md#publish-hotfix --> just checking